### DISCUSSION TITLE
Where I find XML file for my Run if I am using Allure plugin

### DISCUSSION THREAD

**Discussion by Animeshkr29** [Shelex/cypress-allure-plugin#79] [Q&A] [answered]
Hi Dev Team

I want to read XML file once I complete my test run , 
How and where I find XML generated report in Allure folders ?

Thanks 
Animesh

**Comment by Shelex** [comment, marked as answer]
This plugin uses allure2 commons which have results in json file format. Xml, afaik, was used in old allure version.

**Reply by Animeshkr29** [reply]
thanks ,
